Ethernet/Ethernet.py

```diff
@@ -0,0 +1,125 @@
+import socket
+from scapy.all import conf, IFACES
+
+import Arp
+import IP
+from IP import parse_ip
+from Arp import parse_arp
+from Ipv6 import parse_ipv6
+from Vlan import parse_vlan
+import struct
+
+# Ethernet Frame Format: 6 bytes dest, 6 bytes src, 2 bytes ethertype
+ETHERNET_FRAME_FORMAT = "!6s6sH"
+ETH_HEADER_LEN = struct.calcsize(ETHERNET_FRAME_FORMAT)
+BROADCAST_MAC = b'\xff\xff\xff\xff\xff\xff'
+
+ETHERTYPE_PROTOCOL = {
+    0x0800: ("IPv4", parse_ip),
+    0x0806: ("ARP", parse_arp),
+    0x86DD: ("IPv6", parse_ipv6),
+    0x8100: ("VLAN", parse_vlan),
+}
+
+def bytes_to_mac(bytes_form):
+    hexed_bytes = bytes_form.hex()
+    pretty_mac = ""
+    index = 0
+    while index < len(hexed_bytes):
+        pretty_mac += hexed_bytes[index: index + 2] + ":"
+        index += 2
+    return pretty_mac[:len(pretty_mac) - 1]
```

Comment:
By the way, this whole function can be much much much simpler using builtin behavior.
```suggestion
def bytes_to_mac(bytes_form: bytes) -> str:
    return ':'.join(f'{b:02x}' for b in bytes_form)
```